frontend/src/components/UserUpdateRoles.vue

```diff
@@ -105,6 +140,7 @@ export default {
         this.getUserAvailableClientRoles(),
         this.getUserActiveClientRoles()
       ]);
+      // TODO roles that are in effective but not active should not be included in clientRoles
```

Comment:
Are there are consequences to this? Is it inconvenient for the users? Should we "fix" it before going to prod?

Reply:
There's some weird behavior if a role is added to a user directly, and then also provisioned by group or composite. Its shows the role as available to remove/assign but you cant actually do anything with it,

This is kind of a product of how the keycloak apis work (the admin console has the same behavior). It shouldn't really impact anyone with our production use cases.